app/src/main/java/ch/sdp/vibester/profile/ProfileDataProvider.kt

```diff
@@ -1,67 +1,67 @@
-package ch.sdp.vibester.profile
```

Comment:
You explained it in your PR, however I feel the need to just comment here as well, to remind you to uncomment the code here or refactor it later on